<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-10-17T00:03:21 IST" name="Default suite" finished-at="2021-10-17T00:06:52 IST" duration-ms="211496">
    <groups>
    </groups>
    <test started-at="2021-10-17T00:03:21 IST" name="Default test" finished-at="2021-10-17T00:06:52 IST" duration-ms="211496">
      <class name="com.btf.qa.BasicTestFramework.CheckLoginUnsuccessfulMessage">
        <test-method is-config="true" signature="initialize()[pri:0, instance:com.btf.qa.BasicTestFramework.CheckLoginUnsuccessfulMessage@2b4c1d96]" started-at="2021-10-17T00:03:21 IST" name="initialize" finished-at="2021-10-17T00:04:30 IST" duration-ms="69397" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="loginUnsuccessful()[pri:0, instance:com.btf.qa.BasicTestFramework.CheckLoginUnsuccessfulMessage@2b4c1d96]" started-at="2021-10-17T00:04:35 IST" name="loginUnsuccessful" finished-at="2021-10-17T00:06:21 IST" duration-ms="105667" status="FAIL">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[expected:<Invalid credentials> but was:<[[ChromeDriver: chrome on WINDOWS (284aa98d95f5bd61a2daf731d55b7a65)] -> id: spanMessage]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: expected:<Invalid credentials> but was:<[[ChromeDriver: chrome on WINDOWS (284aa98d95f5bd61a2daf731d55b7a65)] -> id: spanMessage]>
at junit.framework.Assert.fail(Assert.java:57)
at junit.framework.Assert.failNotEquals(Assert.java:329)
at junit.framework.Assert.assertEquals(Assert.java:78)
at junit.framework.Assert.assertEquals(Assert.java:86)
at com.btf.qa.BasicTestFramework.CheckLoginUnsuccessfulMessage.loginUnsuccessful(CheckLoginUnsuccessfulMessage.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginUnsuccessful -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.btf.qa.BasicTestFramework.CheckLoginUnsuccessfulMessage@2b4c1d96]" started-at="2021-10-17T00:06:52 IST" name="teardown" finished-at="2021-10-17T00:06:52 IST" duration-ms="422" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.btf.qa.BasicTestFramework.CheckLoginUnsuccessfulMessage -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
